### ISSUE TITLE
ST7735S display controller

### ISSUE THREAD

**Issue by gsauthof** [slmisc/gd32v-lcd#4]
The README.md currently reads:

> The display controller interface is similar to ST7735 [(Datasheet PDF)](https://www.displayfuture.com/Display/datasheet/controller/ST7735.pdf) in case you're looking for low-level documentation.

According to [Sipeed](https://dl.sipeed.com/shareURL/LONGAN/Nano/HDK) the display controller is in fact the [ST7735S][1] and they also provide its [datasheet][1].

[1]: https://dl.sipeed.com/fileList/LONGAN/Nano/HDK/driver%20chip%20ST7735S_V1.5_20150303.pdf

**Comment by tuupola** [comment]
Those are both standard [MIPI DCS ](https://www.mipi.org/specifications/display-command-set) controllers so there is practically no difference.

**Comment by gsauthof** [comment]
Fair enough, I just mentioned it in case you want to make the README more specific / link to the exact matching datasheet.
